src/components/contact.tsx

```diff
@@ -0,0 +1,377 @@
+
+  &:hover {
+    background-color: rgba(57, 151, 156, 0.2);
+  }
+`;
+
+interface StyledSquareProps {
+  dark: boolean;
+}
+
+const InputField = ({ type, id, name, placeholder }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleFocus = () => {
+    setIsActive(true);
+  };
+
+  const handleBlur = (e) => {
+    if (e.target.value === "") {
+      setIsActive(false);
+    }
+  };
+
+  return (
+    <InputWrapper>
+      <StyledInput
+        type={type}
+        id={id}
+        name={name}
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+      />
+      <Label htmlFor={id} className={isActive ? "active" : ""}>
+        {placeholder}
+      </Label>
+    </InputWrapper>
+  );
+};
+
+export function Contact({ isOpen, onClose, dark }) {
+  if (!isOpen) {
+    return null;
+  }
+  const [open, setOpen] = useState(false);
+  const [scope, animate] = useAnimate();
+  // const items = ["Item 1", "Item 2", "Item 3", "Item 4"];
+  const items = [
+    <InputField type="text" id="name" name="name" placeholder="Name" />,
+    <InputField type="email" id="email" name="email" placeholder="Email" />,
+    <InputField type="tel" id="number" name="number" placeholder="Number" />,
+    <InputField
+      type="text"
+      id="message"
+      name="message"
+      placeholder="Message"
+    />,
+    <StyledButton type="submit" className="my-button">
+      Submit
+      <StyledIcon className="styled-icon" />
+    </StyledButton>,
+  ];
+
+  <motion.ul layout>
+    <StyledBox />
+    <StyledContactWrapper />
+    {items.map((item, index) => (
+      <motion.li key={index} layout>
+        {item}
+      </motion.li>
+    ))}
+  </motion.ul>;
+
+  return (
+    <StyledWrapper>
+      <StyledImage dark={dark}>
+        <img src="/logo-icon-white.svg" alt="Logo" width={40} height={40} />
+      </StyledImage>
+      <WrapperContent>
+        <StyledBox>
+          <StyledBoxContentWrapper>
+            <StyledHeading>
+              Get in contact and leave a little description of your request and
+              one of our team will get back to you.
+            </StyledHeading>
+            <StyledForm>
+              <InputField
+                type="text"
+                id="name"
+                name="name"
+                placeholder="Name"
+              />
+              <InputField
+                type="email"
+                id="email"
+                name="email"
+                placeholder="Email"
+              />
+              <InputField
+                type="tel"
+                id="number"
+                name="number"
+                placeholder="Number"
+              />
+              <InputField
+                type="text"
+                id="message"
+                name="message"
+                placeholder="Message"
+              />
+              <StyledButton type="submit" className="my-button">
+                Submit
+                <StyledIcon className="styled-icon" />
```

Comment:
Remove element attribute
```suggestion
                <StyledIcon />
```